doc/api/api_changes.rst

```diff
@@ -407,6 +407,16 @@ Both bindings are disabled if only a subset of the grid lines (in either
 direction) is visible, to avoid making irreversible changes to the figure.
 
 
+Ticklabels are turned off instead of being invisible
+----------------------------------------------------
+
+Internally, :func:`~matplotlib.axis.Axis.set_tick_params` is now used to
+hide tick labels instead of setting the visibility on the tick label objects.
+This improves overall performance and fixes some issues.
+As a consequence, in case those labels ought to be shown, `set_tick_params`
+needs to be used, e.g. `ax.xaxis.set_tick_params(labelbottom=True)`.
```

Comment:
Here and in similar cases below: Is it worth adding the shorter alternative, `ax.tick_params(labelbottom=True)`?  Or giving that instead of the lower-level one? That would be my preference.  I think it is easier for users to work with the single point of entry for manipulating tick-related things.